src/app.js

```diff
@@ -21,13 +22,16 @@ export function app(app) {
     })
   }
 
-  if (document.readyState[0] !== "l") {
-    load()
-  } else {
-    addEventListener("DOMContentLoaded", load)
-  }
+  return emit("loaded", emit, render(state, view))
```

Comment:
or put defer in your script tag if it's in the head, they become equivalent ;)  I really like putting scripts in head so I can just target document.body for render